technic/machines/HV/quarry.lua

```diff
@@ -255,11 +259,9 @@ local function reset_quarry(meta)
 	meta:set_int("finished", 0)
 end
 
-local base_formspec = "size[8,9]"..
+local base_formspec = "size[9,10]"..
```

Comment:
Should these formspec resizes be also based on `has_mcl` state? (any/all of them in other files too)

I've not actually tested these changes so not sure, maybe it is fine.
If anyone happens to have screenshots would be nice to see.

Reply:
MCL if you need these size changes because its inventory is larger.

In other games the player's inventory looks off-center, perhaps you could adjust the size change only for MCL

![imagen](https://github.com/mt-mods/technic/assets/122187904/d7b4ace3-1693-4281-abf7-0dbf92ce27e8)

![imagen](https://github.com/mt-mods/technic/assets/122187904/58d7c7ee-5828-4f37-971a-e834ac79a6ba)

![imagen](https://github.com/mt-mods/technic/assets/122187904/da34ac8b-b67f-4377-a5e8-922dd6240531)

![imagen](https://github.com/mt-mods/technic/assets/122187904/6864ff36-2c85-4b50-9f4d-3b6ee981ed86)

Reply:
fixed, formspec resizes based on MCL state